spec/src/main/asciidoc/metrics_spec.adoc

```diff
@@ -1443,6 +1519,32 @@ MetricRegistry metricRegistry;
 MetricRegistry metricRegistry;
 ----
 
+If the application wishes to use a sub-scope, it can apply the `@SubRegistry` annotation to set a deployment-wide
+sub-registry:
+
+.Example with sub-scope
+[source, java]
+----
+@SubRegistry(name="myApp")
```

Comment:
Make it so that this is only needed if e.g. @ApplicationPath is not set, as otherwise the @ApplicationPath is used